src/vstart.sh

```diff
@@ -677,6 +677,11 @@ EOF
             BLUESTORE_OPTS+="
         bdev ioring = true"
         fi
+        BLUESTORE_OPTS+="
+        bluestore prefer deferred size hdd = 0
+        bluestore throttle bytes = 1073741824
+        crimson_alien_thread_cpu_cores = 25-30,35-40"
```

Comment:
why? please note these settings are not guarded by `if with_crimson`. are these option optimal for all developers' settings?

I only have 8 cores in my testbed.

Reply:
@tchaikov Hi, kefu, I dropped this commit and added a default behaviour for scattering alien threads on to different cores, please take a look, thanks:-)